updatecli/updatecli.d/docker-images/nginx.yaml

```
@@ -11,38 +11,35 @@ scms:
       token: "{{ requiredEnv .github.token }}"
       username: "{{ .github.username }}"
       branch: "{{ .github.branch }}"
+  nginx-gh-mirror:
```

```suggestion
  nginxGithubMirror:
```
I would stick to camelCase for this sources name.